ui/scripts/codemods/linkto-with-on-modifier.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+/* eslint-env node */
+
+// print to console all files that include LinkTo elements using the {{on modifier}}
+module.exports = (env) => {
+  let fileAlerted;
+  return {
+    ElementNode(node) {
+      if (node.tag === 'LinkTo') {
+        if (!fileAlerted) {
+          const usesModifier = node.modifiers.find((modifier) => modifier.path.original === 'on');
+          if (usesModifier) {
+            console.log(env.filePath); // eslint-disable-line
+            fileAlerted = true;
+          }
+        }
+      }
+    },
+  };
+};
```

Comment:
This probably won't be used again but it might be good to keep all of the recast scripts around for examples to help with future codemods.

Reply:
This is great to see, as someone who's never written a codemod!